tools/aws_lab_setup/roles/common/tasks/RedHat.yml

```diff
@@ -12,9 +12,22 @@
   tags:
     - common
 
-- name: RHEL | Stop and disable firewall
+- name: RHEL | Enable specific firewall ports
+  firewalld:
+    immediate: yes
+    permanent: yes
+    service: "{{ item }}"
+    state: enabled
+  with_items: "{{ firewall_ports }}"
+  notify: restart firewalld
+
+- name: RHEL | Start firewalld
```

Comment:
`firewalld` needs to be running _before_ trying to enable ports or it will fail.